comp/core/status/statusimpl/status_test.go

```diff
@@ -951,3 +951,288 @@ func TestFlareProvider(t *testing.T) {
 
 	assert.NotNil(t, flareProvider)
 }
+
+func TestCaseInsensitive(t *testing.T) {
```

Comment:
Great test. I'm wondering if we could simply add a new test case in the `TestGetStatus` to validate the lower functionality works. That way is less code in the which is already quite a lot  

From the new added code we would have to add a new test case in the test `TestGetStatusBySection` to validate we get the section when using the non lower case version.

Reply:
Done in bcc56959f604741e13bb59c59ef0f5ffa92e6e3f.
I added:
- one which test the subsection is returned with different case format on section parameter
- the message return in case of unknown section